src/ui/components/Events/Events.tsx

```diff
@@ -97,6 +97,7 @@ function Events() {
                   currentTime={currentTime}
                   executionPoint={executionPoint!}
                   jumpToCodeAnnotation={jumpToCodeEntriesPerEvent[event.point]}
+                  jumpToCodeLoadingStatus={annotationsStatus === "pending" ? "loading" : "complete"}
```

Comment:
I dunno that it's necessary (maybe TSC is smart enough to check these strings) but "suspense" does export these statuses as constants.